core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/SubmissionComponent.scala

```diff
@@ -216,6 +217,14 @@ trait SubmissionComponent {
             submission.submissionEntity.map(loadSubmissionEntityId(workspaceContext.workspaceIdAsUUID, _))
           ) flatMap { entityId =>
             configIdAction flatMap { configId =>
+              if (configId.isEmpty) {
+                throw new RawlsExceptionWithErrorReport(
+                  ErrorReport(
+                    StatusCodes.BadRequest,
+                    s"Can't find this submission's method config ${submission.methodConfigurationNamespace}/${submission.methodConfigurationName}."
+                  )
+                )
+              }
```

Comment:
without this block, the `configId.get` at line 232 would fail instead, leading to a `NoSuchElement` error which is not helpful to the end user. See also https://github.com/broadinstitute/rawls/pull/1940#discussion_r951857339